.github/workflows/deploy-dev.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         with:
           node-version: 18
       - run: yarn install --frozen-lock-file
-      - run: ./build.sh http://centrapay-docs.dev.s3-website-ap-southeast-1.amazonaws.com
+      - run: ./build.sh http://centrapay-docs.dev.s3-website-ap-southeast-1.amazonaws.com development
```

Comment:
There are other workflows where we run `build.sh` like `validate.yml` where we will want to validate links. I'm wondering if this should default to development and then override for production. Do you think defaulting to `production` is safer because we won't accidentally show draft pages? 🤔

Reply:
Do we want to validate links of draft pages?

Reply:
I would think so - you'll want to know when you first make the page whether you've put in an invalid link.

Reply:
[Vite defaults to development mode](https://vitejs.dev/guide/env-and-mode.html#modes)

Validating links in draft pages gives us a faster feedback loop